bindings/python/edlib.pyx

```diff
@@ -152,3 +154,98 @@ def align(query, target, mode="NW", task="distance", k=-1, additionalEqualities=
+            to itself).
+            This can be useful e.g. when you want edlib to be case insensitive, or if you want certain
+            characters to act as a wildcards.
+            Set to None (default) if you do not want to extend edlib's default equality definition.
+    @return Dictionary with following fields:
+            {string} cigar  Cigar is a standard format for alignment path.
+                Here we are using extended cigar format, which uses following symbols:
+                Match: '=', Insertion to reference: 'I', Deletion from reference: 'D', Mismatch: 'X'.
+                e.g. cigar of "5=1X1=1I" means "5 matches, 1 mismatch, 1 match, 1 insertion (to reference)".
+            {string} NICE  NICE is the indented pairwise alignment. 
+            {string} query_aln  query_aln is the aligned query sequence from the input 'query'
+            {string} match_aln  match_aln is the pairwise alignment from the parsed cigar
+                The symbol '|' represents a read match '='
+                The symbol '.' represents a read mismtach 'X'
+                The space ' ' represents an insertion 'I' or deletion 'D'
+            {string} ref_aln  ref_aln is the aligned reference sequence from the input 'reference'                
+    """
+
+    ## 'path' - find edit distance, start and end locations and alignment path. (Default)
+
+    result = align(query=query, target=reference, mode=mode, task="path", k=k, additionalEqualities=additionalEqualities)
+
+    ## access locations, a list of a set [(start, end)], from yech1990
+    ## ref_pos = locations[0][0]
+    ref_pos = result["locations"][0][0]
+    if ref_pos == None:
+        ref_pos = 0
+    query_pos = 0 ## 0-indexed
+    ref_aln = match_aln = query_aln = ""
+
+    cigar = result["cigar"]
+
+    ## cigar parsing, from yech1990
+    for step, code in re.findall("(\d+)(\D)", cigar):
+        step = int(step)
+        if code == "=":
+            ref_aln += reference[ref_pos : ref_pos + step]
+            ref_pos += step
+            query_aln += query[query_pos : query_pos + step]
+            query_pos += step
+            match_aln += "|" * step
+        elif code == "X":
+            ref_aln += reference[ref_pos : ref_pos + step]
+            ref_pos += step
+            query_aln += query[query_pos : query_pos + step]
+            query_pos += step
+            match_aln += "." * step
+        elif code == "D":
+            ref_aln += reference[ref_pos : ref_pos + step]
+            ref_pos += step
+            query_aln += " " * step
+            query_pos += 0
+            match_aln += " " * step
+        elif code == "I":
+            ref_aln += " " * step
+            ref_pos += 0
+            query_aln += query[query_pos : query_pos + step]
+            query_pos += step
+            match_aln += " " * step
+        else:
+            pass
+
+    alignments = {
+        'cigar': cigar, 
```

Comment:
I would output only those three strings, cigar they already have so no need to return it, and 'NICE' they can construct by joining the three strings if they want.

Reply:
> I would output only those three strings, cigar they already have so no need to return it, and 'NICE' they can construct by joining the three strings if they want.

Ok, that's reasonable. I'll do this.

Reply:
Currently only returning these outputs:

```
    alignments = {
        'target_aln' : target_aln, 
        'match_aln': match_aln,
        'query_aln': query_aln
    }
```